<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>startPlot</class>
 <widget class="QMainWindow" name="startPlot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1655</width>
    <height>923</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StartPlot</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>801</width>
      <height>391</height>
     </rect>
    </property>
    <property name="title">
     <string>加速度x3</string>
    </property>
    <widget class="QCustomPlot" name="customPlot" native="true">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>751</width>
       <height>340</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>801</width>
      <height>391</height>
     </rect>
    </property>
    <property name="title">
     <string>温度</string>
    </property>
    <widget class="QCustomPlot" name="customPlot2" native="true">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>751</width>
       <height>340</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>20</y>
      <width>801</width>
      <height>651</height>
     </rect>
    </property>
    <property name="title">
     <string>欧拉角x3</string>
    </property>
    <widget class="QCustomPlot" name="customPlot3" native="true">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>751</width>
       <height>601</height>
      </rect>
     </property>
    </widget>
   </widget>
   <zorder>groupBox_2</zorder>
   <zorder>groupBox</zorder>
   <zorder>groupBox_3</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1655</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
